docs/vtpm.md

```diff
@@ -0,0 +1,41 @@
+```ts
+ vTPM Object
+ {
+  id: string,
+  is_protected: boolean, // Currently unused by the XAPI, may be used in future.
+  is_unique: boolean // Currently unused by the XAPI, may be used in future.
+ }
+```
+
+- `is_protected`: The TPM is encrypted whenever it's transferred or at stored to disk (and exported)
+- `is_unique`: The TPM contents are exclusive to that VM, that means it has never been cloned (and maybe never exported nor imported)
+
+vTPM features are only enabled if XAPI is recent enough (check for `pool.restrict` or version (>= 8.3)).
+
+- vTPM
+  - At VM creation
+    - A toggle will be displayed in advanced settings under `boot firmware`.
```

Comment:
The exact label of the toggle could be specified. Something like "Attach a vTPM to the VM", or just "vTPM"? We also probably need to define a tooltip for this label - or other means through which you provide explanations to users in XO - to explain what it is.